dist/src/HighchartsReactNative.js

```diff
@@ -90,7 +90,7 @@ export default class HighchartsReactNative extends React.PureComponent {
     render() {
         const scriptsPath = this.state.useCDN ? httpProto.concat(cdnPath) : path;
         const runFirst = `
-           
+           window.data = \"${this.props.data ? this.props.data : null}\";
```

Comment:
Is it enough? It will not be called after chart's update I guess. 

I think that it should be added also in index.html which I sent in the #33 thread. 

Do you agree?

Reply:
I'm not sure that we can send props to index.html I think sending data initially is good enough

Reply:
Only what comes to my mind is set a string like "afterUpdate". Does it make sense or its not useful?

Reply:
Do you mean setting string afterUpdate to global variable when postMessage called in that case we can do set two variables onLoadData and onMessageData differently so they won't effect each other

Reply:
Hi @sebastianbochan what's your decision on PR

Reply:
@manoharglm I agree with you, when someone will request about update, then we will conisder it.

Will merge it tomorrow.

Reply:
Thank you